CodeSniffer/CLI.php

```diff
@@ -1276,4 +1284,23 @@ private function _validateReportWidth($width)
     }//end _validateReportWidth()
 
 
+    /**
+     * Checks if the given command exists on the system
+     *
+     * @param string $cmd The name of the command
+     *
+     * @return bool
+     */
+    private function _commandExist($cmd)
```

Comment:
Please add `s` to the method name (to have `_commandExists`), because it would be grammatically correct.